AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginIntegrationTests/GraphQL/GraphQLWithLambdaAuthIntegrationTests/GraphQLWithLambdaAuthIntegrationTests.swift

```diff
@@ -9,8 +9,8 @@ import XCTest
 import AWSAPIPlugin
 
 @testable import Amplify
-@testable import AmplifyTestCommon
-@testable import AWSAPICategoryPluginTestCommon
+@testable import APIHostApp
+// @testable import AWSAPICategoryPluginTestCommon
```

Comment:
just ignore these changes for now, the remaining tests such as this one has not yet been fixed to build, this PR only gets AWSAPIFunctionalTests to run